proprietary-specs.md

```diff
@@ -5,4 +5,11 @@ ensure that they are up to date with the current version of CloudEvents. That is
 the responsibility of the respective project maintainers.
 
 - [Apache RocketMQ Transport Binding](https://github.com/apache/rocketmq-externals/blob/master/rocketmq-cloudevents-binding/rocketmq-transport-binding.md)
-- [Google Cloud Pub/Sub Protocol Binding](https://github.com/google/knative-gcp/blob/master/docs/spec/pubsub-protocol-binding.md)
\ No newline at end of file
+- [Google Cloud Pub/Sub Protocol Binding](https://github.com/google/knative-gcp/blob/master/docs/spec/pubsub-protocol-binding.md)
+
+Want to add a binding to a proprietary transport?
+
+- Create a specification that follows the structure of the existing binding specifications (e.g. [http binding](http-protocol-binding.md)) - this will help SDK development.
+  - `NOTE :` The specification needs to be publically accessible and manged by the proposing organization.
```

Comment:
perhaps use ** instead of \`  for the NOTE since it shows up kind of weird in the html version